spec/lib/efile/benefits_eligibility_spec.rb

```diff
@@ -470,4 +470,112 @@
       end
     end
   end
+
+  describe "#eitc_amount" do
+    context "client does not qualify for EITC" do
+      before do
+        allow(subject).to receive(:qualified_for_eitc?).and_return false
+      end
+
+      it "returns nil" do
+        expect(subject.eitc_amount).to eq nil
+      end
+    end
+
+    context "when they are qualified for EITC" do
+      let(:earned_income) { 0 }
+      let!(:w2) { create :w2, intake: intake, wages_amount: earned_income }
+
+      before do
+        allow(subject).to receive(:qualified_for_eitc?).and_return true
+        allow_any_instance_of(Dependent).to receive(:qualifying_eitc?).and_return(true)
+      end
+
+      context "when they have 0 EITC-qualifying children" do
+        before do
+          intake.dependents.destroy_all
+        end
+
+        context "their income is $2,724" do
```

Comment:
Non-blocking: I don't understand the purpose of the specific income brackets in these tests. They're OK with me, but I wonder if more description would help. Testing arithmetic is always a bit of a pain, and it's definitely non-blocking since people can change it later as people continuously use/refactor the EITC code.

Reply:
I put all these test cases in because they were what were provided to me by Gabriel in here https://docs.google.com/spreadsheets/d/1SAGDWnAmAHo9Ya-f8SItCfSpma9fwbiT6rAC9LSv33k/edit#gid=0